### ISSUE TITLE
Type de contenu vidéo

### ISSUE THREAD

**Issue by Nicoloye** [Jardin-des-Sciences/website#39]
Voici le formulaire de contribution en l'état aujourd'hui.
Certains de ces champs sont aujourd'hui définis comme des champs de texte, nous nous sommes posé la question de les passer comme champs de taxonomie (des catégories sélectionnables mais auxquelles on peut aussi ajouter de nouveaux éléments), tout dépend de l'aspect redondant des informations qu'ils contiennent.

La question s'est posé en particulier pour "Définition (HD vs. LD)" et le champ "Droits d'auteur (licence)".

Veuillez trouver ci-après une capture d'écran du formulaire de contribution actuel des vidéos.
Je me permettrai d'ouvrir d'autres tickets de ce type pour vous montrer notre avancée et vous permettre de nous faire des retours direct sur vos attentes (modification de l'ordre, ajout d'onglets de présentation, widgets plus intuitifs, etc)
![jds-ct-video](https://cloud.githubusercontent.com/assets/6469188/13179428/3a99a9ea-d724-11e5-9943-4225cdfd6a5b.png)

**Comment by bzg** [comment]
Oui, je suis __pour__ utiliser des champs de taxinomie dès que c'est possible.  Pour HD/LD oui.

**Comment by Nicoloye** [comment]
OK très bien :)

**Comment by Issenmann** [comment]
Je suis ok pour les champs de taxinomie : cela implique-t-il de fournir une liste d'entrées en amont ou l'incrémentation se fait-elle automatiquement à partir des valeurs saisie?

**Comment by bzg** [comment]
@Issenmann Ce serait compliqué d'extraire des taxinomies à la volée, je pense qu'il faudra définir une liste d'autocomplétions possibles en amont.  @Nicoloye est-ce que tu pourras nous dire sur quels champs ces listes seront pertinentes ?

**Comment by Julie-JDS** [comment]
Je reprends cette issue restée ouverte : avons-nous à préciser certains aspects à ce sujet ?
